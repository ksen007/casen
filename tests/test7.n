

def * {|a, b| a*b }

