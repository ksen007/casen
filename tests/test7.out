(DEF * (FUN (PARAM a b) (* a b)))
